CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# How to Contribute to Avalanche
+
+## Setup
+
+To start developing on AvalancheGo, you'll need a few things installed.
+
+- Golang version >= 1.19.6
+- gcc
+- g++
+
+## Issues
+
+### Security
+
+- Do not open up a GitHub issue if it relates to a security vulnerability in AvalancheGo, and instead refer to our [security policy](./SECURITY.md).
+
+### Did you fix whitespace, format code, or make a purely cosmetic patch?
+
+- Changes from the community that are cosmetic in nature and do not add anything substantial to the stability, functionality, or testability of `avalanchego` will generally not be accepted.
```

Comment:
how come this is the case? I think small formatting commits could be helpful?

Reply:
I wanted to avoid unnecessary noise prs (rearranging the readme, reformatting whitespace, etc). I'm down to not include this on the first pass of this if people dont like this